tests/rest/admin/test_user.py

```diff
@@ -445,45 +446,115 @@ def test_requester_is_admin(self):
         self.assertEqual("bob", channel.json_body["displayname"])
         self.assertEqual("email", channel.json_body["threepids"][0]["medium"])
         self.assertEqual("bob@bob.bob", channel.json_body["threepids"][0]["address"])
+        self.assertEqual(True, channel.json_body["admin"])
 
         # Get user
         request, channel = self.make_request(
-            "GET", self.url, access_token=self.admin_user_tok,
+            "GET", self.url_user_1, access_token=self.admin_user_tok,
         )
         self.render(request)
 
         self.assertEqual(200, int(channel.result["code"]), msg=channel.result["body"])
         self.assertEqual("@bob:test", channel.json_body["name"])
         self.assertEqual("bob", channel.json_body["displayname"])
+        self.assertEqual("email", channel.json_body["threepids"][0]["medium"])
+        self.assertEqual("bob@bob.bob", channel.json_body["threepids"][0]["address"])
         self.assertEqual(1, channel.json_body["admin"])
         self.assertEqual(0, channel.json_body["is_guest"])
         self.assertEqual(0, channel.json_body["deactivated"])
 
+        # Create user
```

Comment:
I think it's confusing to have one massive method that tests lots of things, without making it clear what it is testing at each stage. Please could you break it up into smaller cases: for example, one that creates a new user, another that changes the admin flag on an existing user, another that deactivates a user?

Reply:
ok. I have splitted the tests in a few smaler.